themes/ropensci/layouts/blog/single.html

```diff
@@ -50,22 +50,6 @@ <h2 style="text-align: center;">{{ .Title }}</h2>
                 })();
                 </script>
             </div>
```

Comment:
you'd need to remove the if/else statement here since it'd be Discourse only

Reply:
But if there is no topicid it will fail?

Reply:
right, i guess we can leave it then - the plugin for Discourse comments in theory will do the link automatically for us. when I first tried it it lead to duplicate threads being opened in our forum. So i turned that off so we had to do manually. But i haven't tried it again, if it works again, then we can remove this entirely as the forum itself will embed the discussion thread

Reply:
Ok so I'll wait